modules/exploits/unix/webapp/trixbox_ce_endpoint_devicemap_rce.rb

```diff
@@ -149,6 +152,9 @@ def check
 
   def exploit
     res = login(user, pass)
+    unless res
+      return CheckCode::Unknown
```

Comment:
Uh sorry I might not have been clear here, `CheckCode`, and its variants such as `CheckCode::Unknown`, should only be used in the `check` method. This is the `exploit` method, so we should not be returning info about checks here, as the user is exploiting the target at this point. They don't care (in most cases) about checking the target, they just want to exploit it.

Reply:
done

Reply:
This should be fine as login() will fail via raise_error() in all cases but success.